### DISCUSSION TITLE
ByteTrack parameters for high FPS, small, fast-moving objects

### DISCUSSION THREAD

**Discussion by tonyreina** [ultralytics/ultralytics#15925] [Q&A] [answered]
Yolo and the ByteTrack algorithm work fairly well for my use case, but when the objects are moving too quickly through the FOV the ByteTrack algorithm re-identifies the object as a new track. 

I am using a camera with a 50 FPS rate. From what I've read these models are usually tuned to 30 FPS. 

What sort of parameters in ByteTrack (or Yolo) could I tune to be able to maintain the tracking of fast moving objects?

Best.
Very respectfully,
-Tony

**Comment by glenn-jocher** [comment, marked as answer]
@tonyreina hi Tony,

To improve tracking of fast-moving objects, consider adjusting the `track_high_thresh` and `track_low_thresh` parameters in ByteTrack. Increasing the `track_high_thresh` can help maintain object identity across frames. Additionally, ensure your YOLO model's confidence threshold is set appropriately to avoid frequent re-identification.
